lib/derivative_rodeo/storage_targets/sqs_target.rb

```diff
@@ -137,7 +137,17 @@ def queue_name
       ##
       # @api private
       def file_path
-        @file_path ||= file_uri_parts[:file_path]
+        @file_path ||= [file_dir, file_name].join('/')
+      end
```

Comment:
these changes are used in a later pr. they make file_path/file_dir follow the conventions in other targets.